lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1805,7 +1805,10 @@ export class UserAgentApplication {
         acquireTokenAccountKey = Storage.generateAcquireTokenAccountKey(accountId, stateInfo.state);
       }
 
-      if (this.isInteractionRequired(hashParams[Constants.error]) || this.isInteractionRequired(hashParams[Constants.errorDescription])) {
+      const hashErr = hashParams[Constants.error];
+      const hashErrDesc = hashParams[Constants.errorDescription];
+
+      if ((hashErr && this.isInteractionRequired(hashErr)) || (hashErrDesc && this.isInteractionRequired(hashErrDesc))) {
```

Comment:
Could `this.isInteractionRequired` be updated to check the truthiness of whatever is passed to it, that way you don't have to do the truthiness check here?

Reply:
+1, I think we should check in the function rather than beforehand

Reply:
Please review. Added this.